lib/builder-webpack5/src/preview/iframe-webpack.config.ts

```diff
@@ -91,7 +91,9 @@ export default async (options: Options & Record<string, any>): Promise<Configura
     const storiesFilename = 'storybook-stories.js';
     const storiesPath = path.resolve(path.join(workingDir, storiesFilename));
 
-    virtualModuleMapping[storiesPath] = toImportFn(stories);
+    const needPipelinedImport =
+      !!(coreOptions.builder as Webpack5BuilderConfig).options?.lazyCompilation && !isProd;
```

Comment:
Does the builder need to be safe checked here too?

Reply:
If this is a 7.0 change then `builder` will be required (either from the user or provided by `framework`). I believe this change already exists on the `future/base` branch FWIW

Reply:
I copied the code from 

https://github.com/storybookjs/storybook/blob/89830ad76384faeaeb0c19df3cb44232cdde261b/lib/builder-webpack5/src/preview/base-webpack.config.ts#L48-L53

where it isn't being null checked so I assume it is always defined somehow.

Reply:
Alright!